src/main/java/com/lumberjack/Models/Book.java

```diff
@@ -110,4 +125,20 @@ public void setSynopsis(String synopsis) {
         public String getSynopsis() {
             return this.synopsis;
         }
+
+    public int getRating() {
+        return rating;
+    }
+
+    public void setRating(int rating) {
+        this.rating = rating;
+    }
+
+    public ArrayList<String> getLog() {
+        return log;
+    }
+
+    public void setLog(ArrayList<String> log) {
```

Comment:
@chakimanu Aqui o certo é tu pedir por uma String log e adicionar ela no log.
```suggestion
    public void addLog(String log) {
        this.log.add(log);
    }
```